fsdevtools-cli/src/main/java/com/espirit/moddev/cli/CliContextImpl.java

```diff
@@ -149,7 +151,13 @@ public Project getProject() {
 
     private void loadProject(String projectName) {
         if (!StringUtils.isBlank(projectName)) {
-            Project project = connection.getProjectByName(projectName);
+            Project project = null;
+
+            if (StringUtils.isNumeric(projectName)) {
+                project = connection.getProjectById(Long.parseLong(projectName));
```

Comment:
I think this might be kind of confusing, because the parameter is "name" and then used as "id". I suggest you either rename the parameter, or maybe you refactor the method to return a result and add a second method that loads by id in case loading by name didn't work.